04-tutorial.Rmd

```diff
@@ -21,6 +21,83 @@ https://hutchdatascience.org/Choosing_Genomics_Tools/
 ## Basic plots
 
 
+```{r}
+# Installing Your Packages
+if (!require("BiocManager", quietly = TRUE)){
+  install.packages("BiocManager")
+}
+
+
+if (!'TCGAbiolinks' %in% installed.packages()) {
+    BiocManager::install("TCGAbiolinks")
+}
+# Loading packages for R to recognize
+library(BiocManager)
+library(TCGAbiolinks)
+library(dplyr)
+library(DT)
+#browseVignettes("TCGAbiolinks")
+library(TCGAbiolinks)
+
+# Making dataframe to store your data 
+query <- GDCquery(
+  project = "TCGA-BRCA",
+  data.category = "Transcriptome Profiling",
+  data.type = "Gene Expression Quantification",
+  workflow.type = "STAR - Counts"
+)
+
+#GDCquery: This function makes an object for retrieving data from the GDC.
```

Comment:
These kinds of comments are excellent for explaining things! Might be best suited to be text in between code chunks so people stop to read it.